src/core/thread/mle.cpp

```diff
@@ -3154,6 +3156,23 @@ otError Mle::HandleParentResponse(const Message &aMessage, const Ip6::MessageInf
     }
 #endif
 
+    // Share stats with application, if requested.
+    if (mParentResponseCb)
```

Comment:
Should this be moved above the Partition ID checks above (e.g. above L3131)? Or is this intentionally placed after the Partition ID checks?

Reply:
Shouldn't this code block go above L3131 above?

Reply:
Yes. i m looking at my local changes and it does show that the new code is above partition-id check.  When i do a git push it says everything is up-to-date. Not sure whats wrong.Trying to figure out

Reply:
Was my fault.. I never commuted my change and was pushing a old branch. Now its done.